### PR TITLE
brianubyun/Footer

### PR DESCRIPTION
**Pull request by brianubyun**
- Modified src/components/Footer.jsx to have similar design to what was provided
- Possibly change background color with lighter gradient/color along with font size, etc (appearance changes)

image

### CODE REVIEW

**Comment by fardinzam** [comment]
please take a screenshot of the page as well, it makes it easier to see how it looks with everything else

**Comment by brianubyun** [comment]
- here's a ss of the entire page:
image

- npm comment noted
- background is one color

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -1,5 +1,42 @@
 const Footer = () => {
-  return <div>Footer</div>;
+  return (
+    <div
+      className="Footer"
```
Comment (fardinzam) [inline src/components/Footer.jsx:4]: className is how you add CSS styles, it's not an HTML class
We're using TailwindCSS, look at the docs linked in the resources tab on discord to learn more about it
put everything into one div, use <p> tags for the text, and move all the stuff from styles into classname

File: src/components/Footer.jsx [hunk 2]
```diff
@@ -1,11 +1,18 @@
 const Footer = () => {
   return (
     <div className="text-center bg-[#daf0ff] m-5">
-      <p className="mt-4 text-yellow-400 font-bold text-2xl inline-block"> UCR</p>
-      <p className="inline-block ml-2 font-bold text-2xl text-gray-700">Senior Designs</p>
-      <p className="text-xs mb-12">Copyright © ACM UCR - All Rights Reserved.</p>
+      <p className="mt-4 text-yellow-400 font-bold text-2xl inline-block">
+        {" "}
```
Comment (fardinzam) [inline src/components/Footer.jsx:5]: you don't need this

File: src/components/Footer.jsx [hunk 3]
```diff
@@ -1,11 +1,18 @@
...
+        UCR
+      </p>
+      <p className="inline-block ml-2 font-bold text-2xl text-gray-700">
```
Comment (fardinzam) [inline src/components/Footer.jsx:8]: - don't need inline-block
- the Senior Design text isn't gray, you're not limited to the color's I've provided, use an RGB color finder to get the correct text color from the figma file

**Comment by fardinzam** [comment]
> * here's a ss of the entire page:
>   image
> * npm comment noted
> * background is one color

- background color isn't correct
- add more margin to the top and bottom, the footer is not tall enough

**Comment by brianubyun** [comment]
Updated footer component with requirements
image

**Pull request merged**